modules/admin_manual/pages/installation/apps/mediaviewer/index.adoc

```diff
@@ -0,0 +1,38 @@
+= Media Viewer App
+:media-viewer-app-url: https://github.com/owncloud/files_mediaviewer 
+:nodejs-url: https://nodejs.org/en/
+:npm-url: https://www.npmjs.com/get-npm
+:install-imagemagick-url: https://www.tecmint.com/install-imagemagick-on-debian-ubuntu/
+:install-imagemagick-pecl-url: https://www.php.net/manual/en/imagick.installation.php
+
+The {media-viewer-app-url}[Media Viewer App] is a simple viewer for pictures and videos integrated into the files app.
+
+== Requirements
+
+* {nodejs-url}[Node.js]
+* {npm-url}[NPM]
+
+== Installing
+
+Before installing the Media Viewer app, run `npm install` && `npm run build` to build the Media Viewer app's dependencies.
```

Comment:
Required by end-user? Not only possible to install from marketplace without node/npm?

Reply:
This is the admin docs, so I don't see any issue with including it; especially as it's listed in [the repository's README](https://github.com/owncloud/files_mediaviewer).

Reply:
@settermjd no, please remove. Instructions in the README.md are for developers. Marketplace install or install with *.tar are the recommended ways.

Reply:
> This is the admin docs, so I don't see any issue with including it; especially as it's listed in the repository's README.

installing not same as building, README only mentions requirements for building for developers which is developer docs not admin docs.

Reply:
Thank you for the feedback.